<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dance</class>
 <widget class="QWidget" name="dance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="maxNumber">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>40</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>something</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_song">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>song</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="songNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="boyNumber">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>boy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="boyNumber_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_girl">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>girl</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="girlNumber">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>601</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>dance</receiver>
   <slot>ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>dance</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>back()</slot>
 </slots>
</ui>
